Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM python:3.8
+FROM python:3.10
```

Comment:
Is there a reason why this got bumped to 3.10? I use this repository as an example for other things besides basic Flask like using Tensorflow. At the moment, you can't use 3.10 with Tensorflow.

Reply:
Good info. No, there is no particular reason. I happen to be using 3.10 with poetry projects, but not required here.

Reply:
Reverted to python:3.8 as flask 2.2.2 requirement is Python >=3.7